Release.toml

```
@@ -270,3 +270,7 @@ version = "1.19.1"
     "migrate_v1.19.1_aws-control-container-v0-7-8.lz4",
     "migrate_v1.19.1_public-control-container-v0-7-8.lz4",
 ]
+"(1.19.1, 1.20.0)" = [
+    "migrate_v1.20.0_certdog-config-file-v0-1-0.lz4",
+    "migrate_v1.20.0_certdog-service-cfg-v0-1-0.lz4",
+]
```

Nit: missing newline.

```suggestion
]

```